src/API/Endpoints/Reports/Endpoint.php

```diff
@@ -150,4 +150,43 @@ public function authorization_status_code() {
 		return $status;
 
 	}
+
+	/**
+	 * Get cached report
+	 *
+	 * @param WP_REST_Request $request Current request.
+	 */
+	public function get_cached_report( $request ) {
+
+		$query_args = [
+			'start' => $request['start'],
+			'end'   => $request['end'],
+		];
+
+		$cache_key = \Give_Cache::get_key( "api_get_report_{$this->endpoint}", $query_args );
```

Comment:
To avoid having to use the root namespace indicator every time, just put `use \Give_Cache` at the top of the file.

Reply:
Change made, ref: https://github.com/impress-org/givewp/pull/4437/commits/de6824c6f1fe4d87069eaee517f7ec36be7cd658